actpool/actpool.go

```diff
@@ -434,7 +439,12 @@ func (ap *actPool) enqueueAction(sender string, act action.SealedEnvelope, actHa
 // removeConfirmedActs removes processed (committed to block) actions from pool
 func (ap *actPool) removeConfirmedActs() {
 	for from, queue := range ap.accountActs {
-		confirmedState, err := accountutil.AccountState(ap.sf, from)
+		addr, err := address.FromString(from)
```

Comment:
when adding to `ap.accountActs`, it is a valid address, so the address here must be valid, can do:
`addr, _ := address.FromString(from)`

Reply:
done